src/main/webapp/index.html

```diff
@@ -20,7 +21,22 @@
 <body onload="registerServiceWorker(); createStateDropdownList();">
   <main>
     <div id="header">
-      <h1>Census Viz</h1>
+      <h1>Census Viz
+        <div class="tooltip">
+          <i class="material-icons" id="faq">help_outline</i>
+          <p class="tooltiptext">To use Census Viz, start typing in any box. The
```

Comment:
maybe something like "select one of the available options from each drop down list and click the Green Go button. You can also start typing in any drop down to filter the options"

Reply:
That's a good suggestion, thank you. Done